### ISSUE TITLE
download datasets question

### ISSUE THREAD

**Issue by timetail233** [Flanders-Make-vzw/RAISE_LPBF_Laser_benchmark#1]
Hello, I am very interested in the dataset and related tasks you provided. However, when I went to the official website to download, due to unstable network conditions, once the network fluctuated, I had to download again, which prevented me from downloading over 700 GB of files. Do you have any other ways to download datasets?

**Comment by degrave** [comment]
Juelich's Open Science data distribution service seems to be down. I'll contact them so they can fix the root cause.

**Comment by timetail233** [comment]
Thank&nbsp;you&nbsp;very&nbsp;much&nbsp;for&nbsp;your&nbsp;reply.&nbsp;At&nbsp;the&nbsp;same&nbsp;time,&nbsp;I&nbsp;would&nbsp;also&nbsp;like&nbsp;to&nbsp;ask&nbsp;if&nbsp;there&nbsp;are&nbsp;any&nbsp;other&nbsp;ways&nbsp;to&nbsp;download&nbsp;this&nbsp;dataset&nbsp;at&nbsp;present？&nbsp;It&nbsp;is&nbsp;inconvenient&nbsp;to&nbsp;download&nbsp;such&nbsp;a&nbsp;large&nbsp;data&nbsp;set&nbsp;from&nbsp;the&nbsp;web&nbsp;page.&nbsp;Is&nbsp;it&nbsp;possible&nbsp;to&nbsp;split&nbsp;the&nbsp;data&nbsp;set&nbsp;according&nbsp;to&nbsp;the&nbsp;printed&nbsp;parts&nbsp;so&nbsp;that&nbsp;the&nbsp;data&nbsp;set&nbsp;can&nbsp;be&nbsp;downloaded&nbsp;in&nbsp;batches?


---Original---
From: ***@***.***&gt;
Date: Thu, Oct 12, 2023 15:24 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Flanders-Make-vzw/RAISE_LPBF_Laser_benchmark] download datasetsquestion (Issue #1)




 
Juelich's Open Science data distribution service seems to be down. I'll contact them so they can fix the root cause.
 
&mdash;

**Comment by timetail233** [comment]
> Juelich's Open Science data distribution service seems to be down. I'll contact them so they can fix the root cause.

Thank you very much for your reply. At the same time, I would also like to ask if there are any other ways to download this dataset at present？ It is inconvenient to download such a large data set from the web page. Is it possible to split the data set according to the printed parts so that the data set can be downloaded in batches?

**Comment by degrave** [comment]
We don't plan to offer downloads in pieces, since if you just want to check it out, the small sample should do. If you're serious and going to train something, you'll want the complete dataset. But if anyone wants to redistribute the data in pieces, the license allows it as long as you do it non-commercially and with attribution.

**Comment by degrave** [comment]
Juelich is aware of the problem.

**Comment by timetail233** [comment]
> We don't plan to offer downloads in pieces, since if you just want to check it out, the small sample should do. If you're serious and going to train something, you'll want the complete dataset. But if anyone wants to redistribute the data in pieces, the license allows it as long as you do it non-commercially and with attribution.

Thank you very much for your work。I hope to use the complete data, but due to the large size of the data, if there are network issues,I have to download it from scratch, I have tried it many times. So I would like to ask if there are any other ways to download it.

**Comment by degrave** [comment]
The data service is available again.

**Comment by timetail233** [comment]
> The data service is available again.
Thank you for your reply. I really wanted to download the entire file, but after several attempts, I found that I couldn't download such a large file at once. I tried downloading several times, and every time I reached 10g or even 30g, a network error occurred, causing me to have to start downloading the file again. However, the entire file is 700g in size, and I couldn't download this dataset from the official website at all, There is no function to continue downloading after network fluctuations on the official website, I can only download from scratch.I have tried several times, but there is no solution for me

**Comment by timetail233** [comment]
> The data service is available again.

Thank you for your reply. I really wanted to download the entire file, but after several attempts, I found that I couldn't download such a large file at once. I tried downloading several times, and every time I reached 10g or even 30g, a network error occurred, causing me to have to start downloading the file again. However, the entire file is 700g in size, and I couldn't download this dataset from the official website at all, There is no function to continue downloading after network fluctuations on the official website, I can only download from scratch.I have tried several times, but there is no solution for me
